src/Task/Psalm.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace GrumPHP\Task;
+
+use GrumPHP\Runner\TaskResult;
+use GrumPHP\Task\Context\ContextInterface;
+use GrumPHP\Task\Context\GitPreCommitContext;
+use GrumPHP\Task\Context\RunContext;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * Psalm task
+ */
+class Psalm extends AbstractExternalTask
+{
+    const TASK_NAME = 'psalm';
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getName()
+    {
+        return self::TASK_NAME;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getConfigurableOptions()
+    {
+        $resolver = new OptionsResolver();
+        $resolver->setDefaults([
+            'config' => null,
+            'ignore_patterns' => [],
+            'no_cache' => false,
+            'report' => null,
+            'threads' => 1,
```

Comment:
Maybe it is better to set this to `null` and let psalm decide what it uses by default?